lisa/schema.py

```diff
@@ -207,14 +207,43 @@ class Parent:
 
 @dataclass_json()
 @dataclass
-class Extension:
+class ExtensionV1:
```

Comment:
Why can't we fix all the users and get away with legacy stuff here?

Reply:
It's fixing, there are some used in critical pipelines. I will double check if all of them done.